src/VolleyManagement.API/appsettings.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "Logging": {
+    "IncludeScopes": false,
+    "Debug": {
+      "LogLevel": {
+        "Default": "Warning"
+      }
+    },
+    "Console": {
+      "LogLevel": {
+        "Default": "Warning"
+      }
+    }
+  },
+  "ConnectionStrings": {
+    "VolleyManagementEntities": "Server=(localdb)\\mssqllocaldb;Database=VolleyManagement;Trusted_Connection=True;MultipleActiveResultSets=true"
+  },
+  "AppSettings": {
+    "FeatureToggle.IisDeployment": "true",
+    "FeatureToggle.DebugMode": "true"
```

Comment:
Should we have it different in appsettings.Development.json?

Reply:
I will move FeatureToggle.DebugMode into in appsettings.Development.json